src/reducers/notebook-reducer.js

```diff
@@ -43,6 +43,18 @@ function clearHistory(loadedState) {
   /* eslint-enable */
 }
 
+function updateAppMessages(state, messageObj) {
+  const message = Object.assign({}, messageObj)
+  delete message.type
```

Comment:
`message.type` is not in the schema and doesn't appear to exist anywhere else. can this be deleted?

Reply:
sort of ... I think I need to change how the action dispatches so I don't have to do this. This is here because I'm loading an action (which has a `type`) with a all the message attributes.